docs/src/examples/Cloudy_example.md

```diff
@@ -0,0 +1,82 @@
+# Cloudy Example
+
+### What Is This Example About?
+
+This example is based on Cloudy, a microphysics model that simulates how cloud droplets collide and coalesce into larger drops. Collision-coalescence is a crucial process for the formation of rain. 
+
+Cloudy takes two inputs: The initial mass distribution of the cloud droplets, and a so-called collision-coalescence kernel, which specifies the probability that two droplets of masses x and y collide and coalesce in a given time interval. 
+Cloudy simulates the evolution of the initial cloud droplet mass distribution over time, as more and more droplets combine into bigger drops. It does this not by simulating each individual droplet, but by using a "bulk approach", which only keeps track of some number of moments of the cloud droplet mass distribution.
+
+This example shows how Ensemble Kalman methods can be used to learn the parameter of the initial cloud droplet mass distribution from observations of the moments of that mass distribution at a later time. The collision-coalescence kernel is assumed to be known, but one could also learn the parameters of the kernel instead of the parameters of the droplet distribution (or both).
```

Comment:
_can be used to learn the parameter_ parameter or parameters?

Reply:
Following the glossary it should be "parameters", you're right :)